Ret con primary record constructors (#35486)
dotnet/docs: docs/csharp/tour-of-csharp/types.md

The memory occupied by an object is automatically reclaimed when the object is no longer reachable. It's not necessary or possible to explicitly deallocate objects in C#.

:::code language="csharp" source="./snippets/shared/Types.cs" ID="CreatePoints":::

> Many application, or tests for algorithms may need to create multiple `Point` objects. The following class generates a sequence of random points. The number of points is set by the *primary constructor* parameter. The primary constructor parameter `numPoints` is in scope for all members of the class:

Reviewer: ```suggestion
Applications or tests for algorithms might need to create multiple `Point` objects. The following class generates a sequence of random points. The number of points is set by the *primary constructor* parameter. The primary constructor parameter `numPoints` is in scope for all members of the class:
```
